lte/gateway/c/session_manager/StoredState.cpp

```diff
@@ -66,6 +66,14 @@ std::string serialize_stored_session_config(const SessionConfig& stored) {
   marshaled["bearer_id"]         = std::to_string(stored.bearer_id);
   marshaled["qos_info"]          = serialize_stored_qos_info(stored.qos_info);
 
+  std::string common_context;
+  stored.common_context.SerializeToString(&common_context);
```

Comment:
if this is a protobuf, check SerializeAsString (funny name :D) method. It may save you some

Reply:
For the counting algo pr I am doing I also did the same, but eventually I decided to use the regular hard coded way..... I am trying to remember why I decided to use the ugly way, but I think it was due to the default values protos gives when the field is missing.

Reply:
How is `SerializeAsString` different from `SerializeToString` (also protobuf serialization func) ?

Reply:
It gives you the string. You don't need to pass a reference to a string

Reply:
Ohh I see thanks! That shortens it a bit :D